src/components/navigation/HaukiNavigation.css

```diff
@@ -1,3 +1,3 @@
-.navigation-header {
+.navigation-header.navigation-header.navigation-header {
```

Comment:
Not optimal and pretty, can we find a cleaner fix?

Reply:
Alternative options:
```css
 .navigation-header {
  --header-background-color: var(--primary-color) !important; // We need to force override with !important
}
```

```css
 header.navigation-header { // We need more specific selector to override the default hds-styles
  --header-background-color: var(--primary-color) !important; // We need to force override with !important
}
```

Reply:
Would it make sense to make all of the customizations in one place? 
```css
:root {
  --primary-color: #6feed3;
  --header-background-color: var(--primary-color) !important;
}
```

Reply:
> Alternative options:
> 
> ```css
>  .navigation-header {
>   --header-background-color: var(--primary-color) !important; // We need to force override with !important
> }
> ```
> 
> ```css
>  header.navigation-header { // We need more specific selector to override the default hds-styles
>   --header-background-color: var(--primary-color) !important; // We need to force override with !important
> }
> ```

Those would work also, though not sure if they would be cleaner. mm..

Reply:
I think the most cleanest would be to override them all in on place 🤔

Reply:
Good point!

Reply:
- [ ] Override variables in one place